src/main/java/uk/gov/companieshouse/confirmationstatementapi/controller/ShareholderController.java

```diff
@@ -15,7 +13,7 @@
 import java.util.List;
 
 import static uk.gov.companieshouse.confirmationstatementapi.ConfirmationStatementApiApplication.LOGGER;
-import static uk.gov.companieshouse.confirmationstatementapi.utils.Constants.ERIC_REQUEST_ID_KEY;
+import static uk.gov.companieshouse.confirmationstatementapi.utils.Constants.*;
```

Comment:
can you remove the .* import? CH standards mean we have to use individual imports instead of *